src/app/modules/products/components/get-notified-confirmation-modal/get-notified-confirmation-modal.component.ts

```diff
@@ -0,0 +1,27 @@
+import { Component, ViewChild } from "@angular/core";
+import { ModalComponent } from "@shared/components/modal/modal.component";
+
+@Component({
+    selector: "app-get-notified-confirmation-modal",
+    templateUrl: "./get-notified-confirmation-modal.component.html",
+    styleUrls: ["./get-notified-confirmation-modal.component.scss"]
+})
+export class GetNotifiedConfirmationModalComponent {
```

Comment:
not a big fan of this component name. We should change the name of the component to NotificationModal.

Also, we should change the file names accordingly